### DISCUSSION TITLE
取代你的眼球？Meta 取得機械眼球專利

### DISCUSSION THREAD

**Discussion by yi-1111** [1101-GS4719-C/Python-Community-GS4719-C#447] [科技新知]
人造器官又向前邁進了一大步。

美國專利及商標局2021 年 12 月授予 Facebook 母公司 Meta 機械眼球的新專利，並在近日由國外媒體 Business Insider 揭露。
專利內容描述了一種「雙軸機械可轉動的眼球」，內部具有 2 組轉軸來帶動機械眼球轉動，並以外置馬達來提供轉動機械眼球的動力。

不只如此，這項專利還簡要地討論了一種眼球追蹤系統以及一種 AR 裝置，真讓人期待未來技術成熟的那天。

https://technews.tw/2022/01/19/facebook-patented-a-mechanical-eyeball/

**Comment by pekoranoinu** [comment]


**Reply by chia-kai-chang** [reply]
同學不要看到標題就開槍，沒看內文就直接寫評論。

文中提到「雖然 Meta 尚未對外說明對此機械眼球有何規劃，但從專利內容的脈絡似乎可以推敲出一些線索。這項專利本身不斷提及可以配置在機器人上，成為複製人類或動物眼睛的電子動畫裝置，可能還包括類似於人類皮膚的設計來隱藏機械眼球的零件」

**Comment by sean1325418** [comment]
現在不只要有眼球的功能，還要注重美觀不然會被嫌醜

**Comment by lx3211** [comment]
機器人越來越像人了...雖然說距離電影裡演的還差一大截
但感覺那天也不遠了

**Comment by Jay-722** [comment]
機器人很聰明還不夠，之後會不會長得跟人一樣阿。

**Comment by xkllkx** [comment]
之後機器人越來越像人之後，恐怖谷效應會越來越嚴重吧。

**Comment by ghost** [comment]
看到眼球追縱就想到完成不了的專題QAQ

**Comment by d26488399** [comment]
不知道未來有沒有可能成為一種義肢，幫助視障者重見光明。

**Comment by hapiwangy** [comment]
仿生的技術一直是類科技進步的方法之一，這次終於要換仿人了嗎

**Comment by cordydeng** [comment]
這種技術成熟後想必會有很大的幫助，或許雷神索爾用的到XD

**Comment by PoooYen** [comment]
可想而知行動監測裝置或許不是夢

**Comment by henrywen547** [comment]
機器人越來越像人了
不過，機器人也需要像人類一樣擁有瞳孔的構造控制光線保護眼睛?😂還是保護光學裝置?🤔

**Comment by liangyinglly** [comment]
人類感覺要被機器人看透了

**Comment by HaoFish123** [comment]
感謝分享，雖然我無法想像這專利實際上會用在哪，但它聽起來很酷。

**Comment by Jimmy911218** [comment]
這眼球或許能變成跟蹤式的監視器

**Comment by treeann** [comment]
這眼球的想法太厲害了吧!!!

**Comment by sys511613** [comment]
機器人要變得越來越像人類了，期待未來技術的發展

**Comment by alexthebigpp** [comment]
人造器官的發明真的能讓人類醫學進步很多，但後遺症感覺也需要特別注意。

**Comment by yenwei12138** [comment]
連眼球這麼精密的器官都做出來了，搞不好未來真的能用人造器官取代人體器官

**Comment by etin2468** [comment]
機器人的眼睛能更完善的追蹤目標，這樣靈敏度一定能大幅提升

**Comment by Lynn12378** [comment]
機器人越來越像人類很容易引發恐怖谷效應ㄟ
